AISKU/Tests/TestFramework/TestCase.ts

```diff
@@ -1,22 +1,61 @@
 ﻿
 /** Defines a test case */
-class TestCase {
+export class TestCase {
     /** Name to use for the test case */
     public name: string;
 
+    useFakeServer?: boolean;
```

Comment:
Updated the TestClass with extra mocking ability from other projects.

Also added the ability to now emulate an ES3 environment using a simple boolean during class initialization